test/conftest.py

```diff
@@ -76,6 +76,13 @@ def pytest_addoption(parser):
         help="Skip initialization of the test database.",
     )
 
+    # This runs the server in a separate window with a usable console. A developer can add ``import pdb; pdb.set_trace()`` at any point in the bookserver to invoke the debugger and understand what's happening. The same technique also works well in the tests -- stop at a certain point in the test and (if running a Selenium-based test) look at the JavaScript console, or examine local variables in Python, etc.
+    parser.addoption(
```

Comment:
Is this an option that we pass to pytest when we run it?

Reply:
Yes, exactly. It probably needs changes on Unix -- probably prefix the command with `xterm`?